### PR TITLE
(docs) Add Troubleshooting section

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Adds a Troubleshooting section to the README with instructions for developers should they need to fix their dev environments.

### CODE REVIEW

**Comment by ibacher** [comment]
I wish there was a better set of instructions that work consistently than this...

File: README.md [hunk 1]
```diff
@@ -58,6 +58,21 @@ yarn start --sources 'packages/esm-patient-biometrics-app' --sources 'packages/e
 
 Alternatively, you could run `yarn serve` from within the individual packages and then use [import map overrides](http://o3-dev.docs.openmrs.org/#/getting_started/setup?id=import-map-overrides).
 
+## Troubleshooting
+
+If you notice that your local version of the application is not working or that there's a mismatch between what you see locally versus what's in the reference application, you likely have outdated versions of core libraries. To update core libraries, run the following commands:
```
Comment (dkayiwa) [inline README.md:63]: Does `in the reference application` mean `dev3.openmrs.org`?

Comment (denniskigen) [reply README.md:63]: Correct.

Comment (dkayiwa) [reply README.md:63]: Assuming `demo` site does not mean `https://demo.openmrs.org/` 😊

Comment (denniskigen) [reply README.md:63]: I guess dev3 would be the safest bet then :)

Comment (dkayiwa) [reply README.md:63]: How can i say NO? 😊

Comment (denniskigen) [reply README.md:63]: Is that a yes?

Comment (dkayiwa) [reply README.md:63]: Loool, yes. 🤣

**Pull request merged**